src/main/java/com/fasterxml/jackson/databind/introspect/AnnotatedFieldCollector.java

```diff
@@ -124,6 +124,9 @@ private void _addFieldMixIns(Class<?> mixInCls, Class<?> targetClass,
 
     private boolean _isIncludableField(Field f)
     {
+        if (f.isEnumConstant()) {
```

Comment:
Curious as to how/why these are needed? Are they `static` and this is the only way to retain "fields" or... ?

Reply:
Ok yeah, makes sense wrt goals. It's conceptually wrong but... I'll have to think about this. It is sort of clever abuse :)
I like some aspects, and it is safe in the sense that it will not pull in any other static fields. So maybe that is acceptable.